projects/components/src/tabs/navigable/navigable-tab-group.component.scss

```diff
@@ -67,13 +63,14 @@
 }
 
 .tab-link {
-  height: 46px; // plus ink-bar is 48
+  height: auto; // plus ink-bar is 48
```

Comment:
Could you remove the comment - no longer relevant

Reply:
done